doc/developers/contributing.rst

```diff
@@ -952,6 +952,18 @@ The easiest and recommended way to accomplish this is to
 All logic behind estimator parameters,
 like translating string arguments into functions, should be done in ``fit``.
 
+Parameters can be passed to the ``fit`` method. Generally, these parameters
+should be restricted to variables that have the shape ``n_samples``.
```

Comment:
Hmm. Perhaps my use of the shape should be clearer, then. I think the description here http://scikit-learn.org/stable/developers/contributing.html#fitting is a good guide. Perhaps I should use something like 
"...should be restricted to variables that are array with shape=[N] or array-like with shape=[N,D] where N is n_samples..."

What do you think?